### DISCUSSION TITLE
Candy CDSN 2D350PX

### DISCUSSION THREAD

**Discussion by kiraly965** [ofalvai/home-assistant-candy#270] [Device support improvements]
Hello there! I have 3 candy device (oven, washer, dishwasher). The oven and the washing machine were added without any problems, but the washing machine gives the following error:
![kép](https://user-images.githubusercontent.com/35660169/230712174-8663cc86-2d0c-4f88-9b92-1750be237e43.png)
Can someone help me with this?

**Comment by t30** [comment]
I have a dishwasher and I face lot of problem before get it running in hassio.
I am not sure if the problem was the same. At that time I wasn't able also to get control over the app.

I found accidentally a solution...

With the device powered on, try to unplug the AC connection, wait for a few minutes the plug again the AC. (FYI: I do this procedure using the dedicated breaker in the electric panel)

Sometimes I face again this problem (once every 3/4 months) and with the above described procure I solve the issue very quickly

**Reply by manovoskonica** [reply]
I had the same problem too. The workaround is to first make a connection through simplyfi app, after youre done setting that up see on the router if the diswasher is online if it's online the encryption will go through without a problem.
